.github/workflows/test-redis-enterprise.yml

```diff
@@ -0,0 +1,66 @@
+name: RE Tests
+
+on:
+  push:
+    branches: [master, v9]
+  pull_request:
+    branches: [master, v9]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version: [1.21.x]
+        re-build: ["100.0.1-64402"]
+        # go-version: [1.19.x, 1.20.x, 1.21.x]
+
+    steps:
+      - name: Clone Redis EE docker repository
+        run: |
+          git clone https://github.com/RedisLabs/redis-ee-docker.git redis-ee
+      - name: Install Docker-compose
+        run: |
+          DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
+          mkdir -p $DOCKER_CONFIG/cli-plugins
+          curl -SL https://github.com/docker/compose/releases/download/v2.23.3/docker-compose-linux-x86_64 -o $DOCKER_CONFIG/cli-plugins/docker-compose
+      - name: Build cluster
+        env:
+          IMAGE: "redislabs/redis-internal:${{ matrix.re-build }}"
```

Comment:
Maybe instead we can load from dotnet *and* have a comment for why this exists? This way we set off fewer false alarms from automated security tools.

If not dotenv, at least a comment here IMHO.